day_1_ddo/gd_utils.py

```diff
@@ -0,0 +1,99 @@
+# utils.py
+import numpy as np
+import matplotlib.pyplot as plt
+
```

Comment:
For your amusement: I am quite certain that the standard is two empty lines between imports and functions. But again, let's run `black` in the end on all of it :)